packages/shared/chat/sagas.ts

```diff
@@ -189,8 +189,9 @@ function initChatCommands() {
         response = TeleportController.goToMagic().message
       } else if (message.trim().toLowerCase() === 'random') {
         response = TeleportController.goToRandom().message
-      } else if (message.trim().toLowerCase() === 'next') {
-        response = TeleportController.goToNext().message
+      // TODO: this isn't a async or function generator...
+        // } else if (message.trim().toLowerCase() === 'next') {
+        // response = (await TeleportController.goToNext()).message
```

Comment:
To wait for promises outside awaitable contexts, use .catch and .then
```ts
myAsyncFunction().catch(err => sharedLogger.error(err))
```

Reply:
This needs to change other functionality, it's called by a sync callback stored in a object. I need to be careful

Reply:
done